client/my-sites/plans-features-main/index.tsx

```diff
@@ -331,7 +331,6 @@ const PlansFeaturesMain = ( {
 
 	const {
 		isLoading: isTrailMapExperimentLoading,
-		variant: trailMapExperimentVariant,
```

Comment:
As the `variant` value is no longer used, I think we should remove it from the [hook](https://github.com/Automattic/wp-calypso/blob/e9d35d14eab8bdd729d990d1cfd41bbb3c40ae1c/client/my-sites/plans-features-main/hooks/use-experiment-for-trail-map.ts#L48).